lib/serverengine/config_loader.rb

```diff
@@ -56,8 +56,8 @@ def reload_config
     private
 
     def create_logger
-      if logger = @config[:logger]
-        @logger = logger
+      if @config[:logger].is_a?(DaemonLogger)
+        @logger = @config[:logger]
         return
```

Comment:
`create_logger` needs to return a logger. I think this is the bug. So, I think create_logger method needs to be:

```ruby
    def create_logger
      if logger = @config[:logger]
        @logger = logger
      else
        @logger = @logger_class.new(logdev_from_config(@config), @config)
      end
      return @logger
    end
```